packages/instantsearch.js/src/widgets/index/__tests__/index-test.ts

```diff
@@ -1170,6 +1209,36 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/index-widge
       );
     });
 
+    it('calls getWidgetParameters on widgets that depend on recommend', () => {
```

Comment:
Isn't this a duplicate of l. 264?

Reply:
One is ran in `addWidgets` and the other in `init`, hierarchy should show that in reports as it's under two different `describe`

Reply:
I added a precision that it's after 'init'